examples/dscp-app/README.md

```diff
@@ -0,0 +1,194 @@
+## DSCP-APP
+
+This folder contains the files that are needed for the deployment of a sample supply chain application on a Substrate based DLT Network that has been deployed using Hyperledger Bevel. This application uses a node.js API to support communication to the Substrate based dscp-node and an IPFS node via polkadot.js API.
+
+### Folder Structure
+
+***
+
+```
+dscp-app
+|--charts
+   |--certissuer
+   |--dscp-api
+   |--dscp-frontend
+   |--inteli-api
+|--configuration
+```
+
+### Pre-requisites
+
+***
+
+- Substrate network of one or more organizations. A complete supplychain network will have the following organizations:
+  - Supplychain (admin organization)
+  - Carrier
+  - Store
+  - Warehouse
+  - Manufacturer
+- IPFS node
+
+### Authentication
+
+***
+
+The dscp-api uses an Auth0 Machine to Machine API to issue a JSON Web Token for authentication on its endpoints. You will need to create your own Auth0 API, which can be done for free and then set the appropriate environment variables, as shown below.
+
```

Comment:
This is optional and by default is turned off in our newer helm charts.

You can either continue to use the JWKS authentication method or set `auth.type: NONE` and use something else at the cluster level.

Reply:
Thanks for the review David. 👍